singer-io/tap-twitter-ads
# Tdl 18089 make child streams independent of parent stream

# Description of change
- Made child stream independent of parent stream.
- Now, the bookmark for the child stream will also be written as per the parent replication key.
- In the next sync, it will collect only those parent ids which respect the child stream bookmark.
- Unitest case is written for the parent-child bookmark mechanism

**Note**: Removed non-standard test cases as we have separate https://github.com/singer-io/tap-twitter-ads/pull/20 for standard tap tester.
# Manual QA steps
 - Verify that a proper bookmark is written for the child stream.
 - Verify that the child stream respects its own bookmark instead of the parent stream bookmark.
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
